### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by cpanato**
- [x] Check the DNS for the github pages

### CODE REVIEW

**Comment by carlisia** [comment]
Getting a 404 for the moment.

**Comment by cpanato** [comment]
@carlisia maybe becasue this is in draft mode, because I want to make sure the DNS name for the github page will be that one or another

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+// TODO: check the DNS name for the hosting
+```console
+$ helm repo add vmware-tanzu https://vmware-tanzu.github.io/helm-charts
```
Comment (nrb) [inline README.md]: https://vmware-tanzu.github.io/helm-charts/ resolves now.

**Comment by carlisia** [comment]
🎉 

image

**Comment by cpanato** [comment]
ready for review :)

**Pull request merged**